blockly/src/starter/PathfinderStarter.java

```diff
@@ -1,25 +1,31 @@
 package starter;
 
+import client.KeyboardConfig;
+import com.badlogic.gdx.Gdx;
 import contrib.entities.HeroFactory;
 import contrib.level.DevDungeonLoader;
 import contrib.systems.*;
-import contrib.systems.LevelEditorSystem;
 import core.Entity;
 import core.Game;
 import core.components.CameraComponent;
+import core.level.utils.Coordinate;
 import core.systems.LevelSystem;
 import core.utils.Tuple;
 import core.utils.components.path.SimpleIPath;
 import java.io.IOException;
 import java.util.logging.Level;
+import javax.naming.OperationNotSupportedException;
 import level.AiMazeLevel;
 import systems.PathfindingSystem;
 import utils.CheckPatternPainter;
+import utils.pathfinding.BFSPathFinding;
 import utils.pathfinding.DFSPathFinding;
 
 /** This class starts the dungeon Ai level to visualize the DFS and BFS. */
 public class PathfinderStarter {
   private static final boolean DRAW_CHECKER_PATTERN = true;
+  private static boolean noPathfindingSelected = true;
+  private static final boolean noSuSAlgorithmAvailable = true;
```

Comment:
Vlt statt 2 booleans einfach ein Tuple speichern mit 2x PathfindingLogic?
```suggestion
  // Tuple<Original, SuSAlgo>
  private static Tuple<PathFindingLogic, PathFindingLogic> pathfindigs = Tuple.of();
```

Reply:
Also in dem Tupel setze ich dann den jeweils ausgewählten Algorithmus und kann damit überprüfen ob schon einer ausgewählt wurde. Aber wie kann ich dann vorher prüfen ob der zu setzende Algo (eigene implementierung der Sus) schon existiert? dafür hatte ich den `noSusAlgorithmAvailable` erstmal als festen bool und den würde ich ja weiterhin brauchen? Wo könnte ich mir die Info her holen, ob der implementiert wurde?

Reply:
Die Idee ist es ja, dass die SuSAlgo Klassen schon existieren, und dann evtl per Default ein `OperationNotSupportedException` geworfen wird. @AMatutat sonst deine Meinung?

Reply:
> Die Idee ist es ja, dass die SuSAlgo Klassen schon existieren, und dann evtl per Default ein `OperationNotSupportedException` geworfen wird. @AMatutat sonst deine Meinung?

So. 
Ich kann das Ding ja im Hintergrund aufrufen und dann die exception fangen. 

Im devDungeon mit der SpeedPootion macht @Flamtky das auch. Kannst da Mal gucken